sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -129,7 +129,7 @@ def callback(raw_response):
             deserialization_callback=callback,
             polling_method=LROBasePolling(
                 timeout=30,
-                lro_algorithms=StatusPollingGETRequest(),
+                lro_algorithms=TranslationPolling(),
```

Comment:
LROBasePolling is expecting a list here, so we should pass `lro_algorithms=[TranslationPolling()]`